### PR TITLE
Added the possibility to config android-packageName and iosBundleIdentifier

### PR DESCRIPTION
**Pull request by rstraub**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-config",
-  "version": "0.6.2",
+  "version": "0.6.3",
```
Comment (SamVerschueren) [inline package.json]: Don't bump the version. I'll do it when releasing it.

File: index.js [hunk 2]
```diff
@@ -179,6 +179,22 @@ module.exports = (function () {
 	};
 
 	/**
+   * Sets the Android package name of the config file.
+   *
+   * @param {string}    packageName    The android package name.
+   */
+  Config.prototype.setAndroidPackageName = function(packageName) {
+      var regex = new RegExp('[\\w.]+');
+      if (!regex.test(packageName)) {
```
Comment (SamVerschueren) [inline index.js]: Add an empty line above the `if` statement

**Comment by SamVerschueren** [comment]
Thank you very much for this PR! Some extra notes.
1. Indentation is wrong. Indent with tabs instead of spaces. I'd suggest to install an [EditorConfig](http://editorconfig.org/) plugins for your editor of choice.
2. Can you add some extra tests to test these 2 methods?

**Comment by rstraub** [comment]
I made the changes you requested. Although when I actually ran the tests with the npm test script, none of them (even the ones that were defined earlier) seemed to work. I decided to put the tests in the same format as you did, as I am not familiar with the particular tooling setup I wasn't able to find a fix for it.

**Comment by SamVerschueren** [comment]
@rstraub the [linter](https://github.com/sindresorhus/xo) is failing. Have to fix that first before I am able to merge and do a new release. If you have some time and are willing to do another PR for that, be my guest :). I can always help you understand the failures. If not, no problem, I'll have to see when I find some time to fix it.

**Comment by rstraub** [comment]
Sounds like a plan. If I have some spare time I will check into it. 👍

**Comment by SamVerschueren** [comment]
Alright, thanks 👍

File: test/setAndroidPackageName.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import test from 'ava';
```
Comment (SamVerschueren) [inline test/setAndroidPackageName.js:1]: Rename the file to `set-android-package-name.js`

File: test/setIOSBundleIdentifier.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import test from 'ava';
```
Comment (SamVerschueren) [inline test/setIOSBundleIdentifier.js:1]: Rename the file to `set-ios-bundle-identifier.js`

**Review by SamVerschueren — CHANGES_REQUESTED** [review]
Files are wrongly named. Can you also rebase on master? It should hopefully solve the issues.

**Comment by SamVerschueren** [comment]
Ping @rstraub Still interested in completing this?

**Pull request merged**